jsapp/js/components/common/badge.module.scss

```diff
@@ -16,6 +16,10 @@ $badge-font-l: sizes.$x14;
   text-align: center;
 }
 
+.label {
+  white-space: nowrap;
+}
+
```

Comment:
Note that when both a label and an icon are included, a wrap point may still appear in between them:

<img src="https://github.com/kobotoolbox/kpi/assets/3514715/afb3fda1-c8e4-4bb6-bc91-fc34e4673cbb" width=256>

Reply:
Right, I pushed a better solution